src/fuzz.rs

```diff
@@ -126,11 +154,270 @@ pub fn partial_token_set_ratio(s1: &str, s2: &str, force_ascii: bool, full_proce
+                s9a: "{a",
+                s10: "a{",
+                s10a: "{b",
+                cirque_strings: &[
+                    "cirque du soleil - zarkana - las vegas",
+                    "cirque du soleil ",
+                    "cirque du soleil las vegas",
+                    "zarkana las vegas",
+                    "las vegas cirque du soleil at the bellagio",
+                    "zarakana - cirque du soleil - bellagio",
+                ],
+                baseball_strings: &[
+                    "new york mets vs chicago cubs",
+                    "chicago cubs vs chicago white sox",
+                    "philladelphia phillies vs atlanta braves",
+                    "braves vs mets",
+                ],
+            }
+        }
+    }
+
+    #[test]
+    fn test_equal() {
+        let f = Fixture::new();
+        assert_eq!(fuzz::ratio(f.s1, f.s1a), 100);
+        assert_eq!(fuzz::ratio(f.s8, f.s8a), 100);
+        assert_eq!(fuzz::ratio(f.s9, f.s9a), 100);
+        // TODO: These are from the Logan's tests, so testing these scores may not be valid.
+        assert_eq!(fuzz::ratio("hello test", "hello world"), 57);
+        assert_eq!(fuzz::ratio("hello test", "hello worlasdfasd"), 52);
+    }
+
+    #[test]
+    fn test_case_insensitive() {
+        let f = Fixture::new();
+        assert_ne!(fuzz::ratio(f.s1, f.s2), 100);
+        assert_eq!(
+            fuzz::ratio(
+                utils::full_process(f.s1, false).as_str(),
+                utils::full_process(f.s2, false).as_str()
+            ),
+            100
+        );
+    }
+
+    #[test]
+    fn test_partial_ratio() {
+        let f = Fixture::new();
+        assert_eq!(fuzz::partial_ratio(f.s1, f.s3), 100);
+        // TODO: These are from the Logan's tests, so testing these scores may not be valid.
+        assert_eq!(fuzz::partial_ratio("hello", "hello world"), 100);
+    }
+
+    #[test]
+    fn test_token_sort_ratio() {
+        let f = Fixture::new();
+        assert_eq!(fuzz::token_sort_ratio(f.s1, f.s1a, true, true), 100);
+        // TODO: These are from the Logan's tests, so testing these scores may not be valid.
+        assert_eq!(
+            fuzz::token_sort_ratio("hello world", "world hello", false, false),
+            100
+        );
+    }
+
+    #[test]
+    fn test_partial_token_sort_ratio() {
+        let f = Fixture::new();
+        assert_eq!(fuzz::partial_token_sort_ratio(f.s1, f.s1a, true, true), 100);
+        assert_eq!(fuzz::partial_token_sort_ratio(f.s4, f.s5, true, true), 100);
+        assert_eq!(
+            fuzz::partial_token_sort_ratio(f.s8, f.s8a, true, false),
+            100
+        );
+        assert_eq!(fuzz::partial_token_sort_ratio(f.s9, f.s9a, true, true), 100);
+        assert_eq!(
+            fuzz::partial_token_sort_ratio(f.s9, f.s9a, true, false),
+            100
+        );
+        assert_eq!(
+            fuzz::partial_token_sort_ratio(f.s10, f.s10a, true, false),
+            100
```

Comment:
```suggestion
            50
```

Reply:
Great catch, [source](https://github.com/seatgeek/fuzzywuzzy/blob/master/test_fuzzywuzzy.py#L135). Adding this suggestion.